### PR TITLE
[Feature-5498][JsonSplit-api] Refactor of dolphin-scheduler-ui/dag

### PR DESCRIPTION
**Pull request by wangyizhi1**
#5498
refactor of dolphin-scheduler-ui dag module

### CODE REVIEW

File: dolphinscheduler-ui/src/js/conf/home/pages/dag/_source/dag.vue [hunk 1]
```diff
@@ -15,890 +15,478 @@
...
+        return tasks.map(task => {
+          const preTask = preTaskMap[task.code]
+          return {
+            name: preTask ? preTask.edgeLabel : '',
+            preTaskCode: preTask ? preTask.sourceId : 0,
+            preTaskVersion: preTask ? tasksMap[preTask.sourceId].version : 0,
+            postTaskCode: task.code,
+            postTaskVersion: tasksMap[task.code].version || 0,
+            conditionType: 0,
+            conditionParams: {}
```
Comment (wen-hemin) [inline dolphinscheduler-ui/src/js/conf/home/pages/dag/_source/dag.vue:392]: Here we need to consider the parameters of conditional branches

File: dolphinscheduler-ui/src/js/conf/home/pages/dag/_source/images/fullscreen.svg [hunk 2]
```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg width="19px" height="19px" viewBox="0 0 19 19" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+    <title>全屏</title>
```
Comment (wen-hemin) [inline dolphinscheduler-ui/src/js/conf/home/pages/dag/_source/images/fullscreen.svg]: Describe internationalization

**Review by wen-hemin — COMMENTED** [review]
Please check

File: dolphinscheduler-ui/src/js/conf/home/pages/dag/_source/dag.vue (hunk 1 above)
Comment (lenboo) [reply dolphinscheduler-ui/src/js/conf/home/pages/dag/_source/dag.vue:392]: > Here we need to consider the parameters of conditional branches

i think he will improve this later.

Comment (wangyizhi1) [reply dolphinscheduler-ui/src/js/conf/home/pages/dag/_source/dag.vue:392]: Thank you for your review!

``conditionType`` and ``conditionParams`` are reserved fields and have no practical meaning at present.
The parameters of the CONDITIONS task are passed in taskDefinitionJson:
```
[{
  "code": 703646130176,
  "name": "test",
  "description": "",
  "taskType": "CONDITIONS",
  "taskParams": {
    "conditionResult": {
      "successNode": [
        "shell-1"
      ],
      "failedNode": [
        "shell-2"
      ]
    }
  },
  "flag": "YES",
  "taskPriority": "MEDIUM",
  "workerGroup": "default",
  "failRetryTimes": "0",
  "failRetryInterval": "1",
  "timeoutFlag": "CLOSE",
  "timeoutNotifyStrategy": "",
  "timeout": 0,
  "delayTime": "0"
}, ... ]
```
I will add a comment here.

File: dolphinscheduler-ui/src/js/conf/home/pages/dag/_source/images/fullscreen.svg (hunk 2 above)
Comment (wangyizhi1) [reply dolphinscheduler-ui/src/js/conf/home/pages/dag/_source/images/fullscreen.svg]: SVG icons cause a lot of problems, such as  internationalization, license header ...
I'm going to convert them to PNG.

**Comment by brave-lee** [comment]
pls fix ut

**Review by brave-lee — COMMENTED** [review]
+1

**Review by lenboo — APPROVED** [review]
+1

**Pull request merged**